command/init.go

```diff
@@ -541,15 +541,38 @@ func (c *InitCommand) getProviders(earlyConfig *earlyconfig.Config, state *state
 			}
 		},
 		FetchPackageSuccess: func(provider addrs.Provider, version getproviders.Version, localDir string, authResult *getproviders.PackageAuthenticationResult) {
-			var warning string
+			var (
+				keyID   string
+				warning string
+			)
 			if authResult != nil {
+				if authResult.ThirdPartySigned() {
+					keyID = authResult.KeyID
+				}
 				warning = authResult.Warning
 			}
+			if keyID != "" {
+				keyID = c.Colorize().Color(fmt.Sprintf(" [reset][bold]%s[reset]", keyID))
+			}
 			if warning != "" {
 				warning = c.Colorize().Color(fmt.Sprintf("\n  [reset][yellow]Warning: %s[reset]", warning))
 			}
 
-			c.Ui.Info(fmt.Sprintf("- Installed %s v%s (%s)%s", provider.ForDisplay(), version, authResult, warning))
+			c.Ui.Info(fmt.Sprintf("- Installed %s v%s (%s%s)%s", provider.ForDisplay(), version, authResult, keyID, warning))
```

Comment:
It's not clear to me how  a user is going to parse this message without anything indicating what each field is - ie how will they know that what they are seeing is a keyID vs some other random ID ?
Can you include a screenshot or test that shows the UX?